x-pack/test/apm_api_integration/common/synthtrace_es_client_service.ts

```diff
@@ -11,5 +11,6 @@ import { InheritedFtrProviderContext } from './ftr_provider_context';
 export async function synthtraceEsClientService(context: InheritedFtrProviderContext) {
   const es = context.getService('es');
 
-  return new apm.ApmSynthtraceEsClient(es, createLogger(LogLevel.info));
+  const forceDataStreams = false;
+  return new apm.ApmSynthtraceEsClient(es, createLogger(LogLevel.info), forceDataStreams);
```

Comment:
At this point it might make sense to use a single argument with named keys rather than three separate ones.

Reply:
Cool happy to :smile:  Is there an official style guide recommendation for the under/over to move to `object` as args? I tend to consider `>3` args.